stdlib/LinearAlgebra/src/generic.jl

```diff
@@ -1897,3 +1897,46 @@ end
 
 normalize(x) = x / norm(x)
 normalize(x, p::Real) = x / norm(x, p)
+
+"""
+    copytotri!(uplo, A, B) -> B
+
+Copies a triangular part of a matrix `A` to another matrix `B`.
+`uplo` specifies the part of the matrix `A` to be copied to `B`.
+Set `uplo = 'L'` for the lower triangular part or `uplo = 'U'
+for the upper triangular part.
+
+!!! compat "Julia 1.11"
+    `copytotri!` requires at least Julia 1.11.
+
+# Examples
+```jldoctest
+julia> A = [1 2 ; 3 4];
+
+julia> B = [0 0 ; 0 0];
+
+julia> copytotri!('L', A, B)
+2×2 Matrix{Int64}:
+ 1  0
+ 3  4
+```
+"""
+function copytotri!(uplo::AbstractChar, A::AbstractMatrix, B::AbstractMatrix)
```

Comment:
This should perhaps mimick `copyto!` closer, so mutated arguments go first:

```suggestion
function copytotri!(B::AbstractMatrix, uplo::AbstractChar, A::AbstractMatrix)
```